libpimeval/src/pimSim.h

```diff
@@ -133,6 +133,10 @@ class pimSim
   ~pimSim();
   pimSim(const pimSim&) = delete;
   pimSim operator=(const pimSim&) = delete;
+  void init();
+  void init(const std::string& simConfig);
```

Comment:
The parameter name is confusion and does not reflect it needs to be file contents. Can it just be file name string?

Reply:
Thanks. Renamed this variable to `simConfigFileConetnt`.